### PR TITLE
docs: vignette on the rationale for the algorithm

### PR DESCRIPTION
**Pull request by lwjohnst86**
This was mostly copied over from @Aastedet thesis page, and revised to fit better as a vignette and to have a lighter/less academic narrative.

### CODE REVIEW

File: vignettes/references.bib [hunk 1]
```diff
@@ -11,4 +11,145 @@ @Article{Isaksen2023
...
+  year   = {2018},
+  url    = {https://sundhedsdatastyrelsen.dk/-/media/sds/filer/registre-og-services/nationale-sundhedsregistre/doedsaarsager-og-biologisk-materiale/laboratoriedatabasen/dokumentation-af-labdatabasens-forskertabel.pdf},
+}
+
+@Misc{SDHDA2022lastaccessed7Feb2023,
+  author = {{Danish Health Data Authority}},
+  title  = {{Algoritmer for udvalgte kroniske sygdomme og svære psykiske lidelser [English: Algorithms for selected chronic diseases and serious mental illnesses]}},
+  year   = {2022},
+  url    = {https://www.esundhed.dk/-/media/Files/Publikationer/Emner/Operationer-og-diagnoser/Udvalgte-kroniske-sygdomme-svaere-psykiske-lidelser/Algoritmer-for-Udvalgte-Kroniske-Sygdomme-og-svre-psykiske-lidelser-RUKS-2022.ashx},
+}
```
Comment (Aastedet) [inline vignettes/references.bib:153]: Same deal here, This link is dead.

Currently, the working link is: https://www.esundhed.dk/-/media/Files/Publikationer/Emner/Operationer-og-diagnoser/Udvalgte-kroniske-sygdomme-svaere-psykiske-lidelser/Algoritmer-for-Udvalgte-Kroniske-Sygdomme-og-svre-psykiske-lidelser.ashx

Going by the name of this URL, this link might stay alive even if they update the documentation (there's no year-reference in the URL). I guess we should update the links to this one?

File: vignettes/references.bib [hunk 2]
```diff
@@ -11,4 +11,145 @@ @Article{Isaksen2023
...
+  url     = {https://www.ncbi.nlm.nih.gov/pubmed/25565889
+https://www.ncbi.nlm.nih.gov/pmc/PMC4274151/},
+}
+
+@Misc{DHDA2016,
+  author = {{Danish Health Data Authority}},
+  title  = {{Algoritmer for udvalgte kroniske sygdomme og svære psykiske lidelser [English: Algorithms for selected chronic diseases and serious mental illnesses]}},
+  year   = {2016},
+  url    = {https://www.esundhed.dk/-/media/Files/Dokumentation/Register-for-Udvalgte-Kroniske-Sygdomme-og-Svaere-Psykiske-Lidelser/29_Algoritmer_for_RUKS.ashx},
+}
```
Comment (Aastedet) [inline vignettes/references.bib:139]: The links to previous RSCD/RUKS documentation are killed every time they update the documentation, so the links to DHDA2016 and SDHDA2022lastaccessed7Feb2023 are now dead. The original documentation is inaccessible to the public and all you have to go by is the "revision log" table in the current word document, which is not really sufficient for reproducibility.

**Review by Aastedet — CHANGES_REQUESTED** [review]
Change DHDA links in the references to the currently working one? (https://www.esundhed.dk/-/media/Files/Publikationer/Emner/Operationer-og-diagnoser/Udvalgte-kroniske-sygdomme-svaere-psykiske-lidelser/Algoritmer-for-Udvalgte-Kroniske-Sygdomme-og-svre-psykiske-lidelser.ashx)

File: vignettes/rationale.Rmd [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+purposes, which provides (authorized) researchers a set of common,
+extensive data sources to use for studies. Any researcher associated
+with an approved Danish research institute (mainly Danish universities)
+can apply for access, but fees and conditions apply.
+
+Register data is generally accessed and processed by approved
+researchers on remote servers operated by Statistics Denmark and the
+Danish Health Data Authority. The same raw data used by all researchers
+and the common virtual working environment potentially enables
+reproducible research. This means that any data processing workflow
```
Comment (signekb) [inline vignettes/rationale.Rmd:46]:
```suggestion
Danish Health Data Authority. The same raw data used by all 
researchers, coupled with a common virtual working environment, 
has the potential to enable reproducible research. This means that 
any data processing workflow
```

File: vignettes/rationale.Rmd [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+could be transferable and usable between research projects if the
```
Comment (signekb) [inline vignettes/rationale.Rmd:47]:
```suggestion
could be transferable and reusable between research projects if the
```

File: vignettes/rationale.Rmd [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+researcher with access to the necessary register data to dynamically
+identify a study population of individuals with diabetes for their
+research needs [@Dima2017].
+
+### Current Danish register-based diabetes classifiers
+
+In Denmark, the first resource readily available to researchers to
+identify diabetes cases through register data was the National Diabetes
+Register, which was established in 2006 [@Carstensen2011] but then
+discontinued in 2012.
```
Comment (signekb) [inline vignettes/rationale.Rmd:62]:
```suggestion
In Denmark, the National Diabetes Register, established in 2006, was the first resource readily
 available to researchers to use for identifying diabetes cases through register data [@Carstensen2011] .
However, it was discontinued in 2012.
```

File: vignettes/rationale.Rmd [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+
+The next resource is the Register of Selected Chronic Diseases (RSCD),
```
Comment (signekb) [inline vignettes/rationale.Rmd:64]: Add source for RSCD? 
The Danish Health Data Authority. The register of selected chronic diseases; 2014. Available from: https://www.esundhed.dk/Dokumentation/DocumentationExtended?id=29. Accessed November 25, 2022.

File: vignettes/rationale.Rmd [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+which was launched in 2014. It is currently the only publicly available
+resource to identify diabetes cases through Danish register data (by
+application to the Danish Health Data Authority).
+
+## Challenges in current classifiers
+
+General-purpose registers and other administrative databases often
+provide the basis of diabetes epidemiology, but they rarely contain
+validated diabetes-specific data, which may introduce bias in studies
```
Comment (signekb) [inline vignettes/rationale.Rmd:73]: Maybe relevant to note what kinds of biases?

**Review by signekb — APPROVED** [review]
Very nice! I have added a couple of suggestions. Nothing big :)

File: vignettes/rationale.Rmd (hunk 6 above)
Comment (lwjohnst86) [inline vignettes/rationale.Rmd:64]:
```suggestion
The next resource is the [Register of Selected Chronic Diseases](https://www.esundhed.dk/Dokumentation/DocumentationExtended?id=29) (RSCD),
```

File: vignettes/references.bib [hunk 8]
```diff
@@ -11,4 +11,145 @@ @Article{Isaksen2023
...
+  type    = {Journal Article},
+  url     = {https://www.ncbi.nlm.nih.gov/pubmed/25565889
+https://www.ncbi.nlm.nih.gov/pmc/PMC4274151/},
+}
+
+@Misc{DHDA2016,
+  author = {{Danish Health Data Authority}},
+  title  = {{Algoritmer for udvalgte kroniske sygdomme og svære psykiske lidelser [English: Algorithms for selected chronic diseases and serious mental illnesses]}},
+  year   = {2016},
+  url    = {https://www.esundhed.dk/-/media/Files/Dokumentation/Register-for-Udvalgte-Kroniske-Sygdomme-og-Svaere-Psykiske-Lidelser/29_Algoritmer_for_RUKS.ashx},
```
Comment (lwjohnst86) [inline vignettes/references.bib:138]:
```suggestion
  url    = {[esundhed.dk/-/media/Files/Publikationer/Emner/Operationer-og-diagnoser/Udvalgte-kroniske-sygdomme-svaere-psykiske-lidelser/Algoritmer-for-Udvalgte-Kroniske-Sygdomme-og-svre-psykiske-lidelser.ashx](https://www.esundhed.dk/-/media/Files/Publikationer/Emner/Operationer-og-diagnoser/Udvalgte-kroniske-sygdomme-svaere-psykiske-lidelser/Algoritmer-for-Udvalgte-Kroniske-Sygdomme-og-svre-psykiske-lidelser.ashx)},
```

File: vignettes/references.bib (hunk 1 above)
Comment (lwjohnst86) [inline vignettes/references.bib:153]:
```suggestion
```

This isn't even used in the document, I must have accidentally added it from the other bib file.

**Pull request merged**